src/DependentField.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from 'proptypes';
+import { connect } from 'react-redux';
+import get from 'lodash.get';
+import set from 'lodash.set';
+import FormField from 'admin-on-rest/lib/mui/form/FormField';
+
+const getValue = (value, path) => {
```

Comment:
Since you use it in DependentInput too, you should move it to another file (DRY)